Inventory-service/src/main/java/com/ahmedshaban/inventoryservice/dto/InventoryResponse.java

```diff
@@ -0,0 +1,15 @@
+package com.ahmedshaban.inventoryservice.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class InventoryResponse {
```

Comment:
good to add in dto and the same in another one but the best practise is to add it in shared multi module